site/src/component/ChangelogModal/ChangelogModal.scss

```
@@ -0,0 +1,37 @@
+.changelog-modal {
+  .modal-content {
+    background-color: var(--overlay2);
+    border-radius: var(--border-radius);
+    padding: 1rem;
+  }
+
+  .modal-header {
+    align-items: center;
+    border-bottom: none;
+  }
+
+  h2 {
+    margin-bottom: 0;
+    font-size: 1.8rem;
+    font-weight: 600;
+  }
+
+  .modal-body {
+    font-size: 1.2rem;
+    margin: 0;
+  }
+
+  .modal-img {
+    margin: 1rem 0;
+    max-width: 70vw;
+    max-height: 50vh;
+    object-fit: contain;
+  }
```

Close button font size can be adjusted to make it bigger/match the year modal.
```suggestion
  }
  
  button.close {
    font-size: 32px;
  }
```